### PR TITLE
Dtd 273

### PR DESCRIPTION
**Pull request by rogerball**

### CODE REVIEW

**Review by aasheishmody — APPROVED** [review]

File: src/test/scala/uk/gov/hmrc/perftests/debttransformation/requests/ifs/SuppressionsRequests.scala [hunk 1]
```diff
@@ -211,8 +249,10 @@ object SuppressionsRequests extends ServicesConfiguration {
       .headers(requestHeaders)
       .body(StringBody(ifsOverlappingSuppression))
       .check(status.is(200))
+      .check(regex("suppressionApplied").find(0))
 
-  val ifsTwoDutiesTwoPaymentsOnSameDay  =
+  println("Simulation is about to start!£££££££££££££££££££££££")
```
Comment (thadeusssewanyana) [inline src/test/scala/uk/gov/hmrc/perftests/debttransformation/requests/ifs/SuppressionsRequests.scala:254]: Do we still need this print statement?

File: src/test/scala/uk/gov/hmrc/perftests/debttransformation/requests/ifs/SuppressionsRequests.scala [hunk 2]
```diff
@@ -64,34 +60,75 @@ object SuppressionsRequests extends ServicesConfiguration {
...
+  }
+
+  def deleteSuppressionRules(): StandaloneWSResponse = {
+    val baseUri = s"$interestForecostingApiUrl/suppression-rules"
+    val headers = Map(
+      "Authorization" -> s"Bearer $bearerToken",
+      "Content-Type"  -> "application/json",
+      "Accept"        -> "application/vnd.hmrc.1.0+json"
+    )
+    print("url ************************" + baseUri)
```
Comment (thadeusssewanyana) [inline src/test/scala/uk/gov/hmrc/perftests/debttransformation/requests/ifs/SuppressionsRequests.scala:129]: Do we still need this print statement?

File: src/test/scala/uk/gov/hmrc/perftests/debttransformation/requests/ifs/SuppressionsRequests.scala [hunk 3]
```diff
@@ -64,34 +60,75 @@ object SuppressionsRequests extends ServicesConfiguration {
...
+  }
+
+  def postSuppressionRules(): StandaloneWSResponse = {
+    val baseUri = s"$interestForecostingApiUrl/suppression-rules/1/suppression-rule"
+    val headers = Map(
+      "Authorization" -> s"Bearer $bearerToken",
+      "Content-Type"  -> "application/json",
+      "Accept"        -> "application/vnd.hmrc.1.0+json"
+    )
+    print("url ************************" + baseUri)
```
Comment (thadeusssewanyana) [inline src/test/scala/uk/gov/hmrc/perftests/debttransformation/requests/ifs/SuppressionsRequests.scala:118]: Do we still need this print statement?

File: src/test/scala/uk/gov/hmrc/perftests/debttransformation/requests/ifs/SuppressionsRequests.scala [hunk 4]
```diff
@@ -64,34 +60,75 @@ object SuppressionsRequests extends ServicesConfiguration {
...
-      .headers(requestHeaders)
-      .check(status.is(200))
+  def postOpenEndedSuppressionData() = {
+    val baseUri = s"$interestForecostingApiUrl/suppressions/2/suppression"
+    val headers = Map(
+      "Authorization" -> s"Bearer $bearerToken",
+      "Content-Type"  -> "application/json",
+      "Accept"        -> "application/vnd.hmrc.1.0+json"
+    )
+    print("url ************************" + baseUri)
```
Comment (thadeusssewanyana) [inline src/test/scala/uk/gov/hmrc/perftests/debttransformation/requests/ifs/SuppressionsRequests.scala:96]: Do we still need this print statement?

**Review by thadeusssewanyana — APPROVED** [review]

**Pull request merged**